testsuite/tests/kuadrant/authorino/operator/http/conftest.py

```diff
@@ -0,0 +1,38 @@
+"""Conftest for all tests requiring custom deployment of Authorino"""
+import pytest
+
+from testsuite.objects import Authorization
+from testsuite.httpx import HttpxBackoffClient
+
+
+# pylint: disable=unused-argument
+@pytest.fixture(scope="module")
+def authorization(authorization, wildcard_domain, blame, openshift, rhsso_service_info, module_label) -> Authorization:
+    """In case of Authorino, AuthConfig used for authorization"""
+    authorization.remove_all_hosts()
+    authorization.add_host(wildcard_domain)
+    resp = {'name': 'another-json-returned-in-a-header',
+            'wrapperKey': 'x-ext-auth-other-json',
+            'json': {'properties': [
+                {'name': 'propX', 'value': 'valueX'}
+            ]}}
+    authorization.add_response(response=resp)
+    return authorization
+
+
+@pytest.fixture(scope="module")
+def client_http_auth(authorization, authorino_route):
```

Comment:
Can you rename this to simply `client`?

Reply:
And that's exactly why I want this to be renamed. It acts exactly as a client and provides the same functionality. The original fixture doesnt work in this context as there is no envoy here, so we can safely replace it by this one which does work and is used for the same purpose.

Reply:
ok, if you think this is a good idea, then I rename it.